client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -1,155 +1,102 @@
-import { fetchAutomatedTransferStatus } from 'calypso/state/automated-transfer/actions';
-import {
-	isFetchingAutomatedTransferStatus,
-	getAutomatedTransferStatus,
-} from 'calypso/state/automated-transfer/selectors';
-import { getIsPluginInstalledDuringPurchase } from 'calypso/state/marketplace/purchase-flow/selectors';
+import { getPluginOnSite } from 'calypso/state/plugins/installed/selectors';
+import { getSiteAdminUrl } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
-import './style.scss';
-
-const MarketplaceThankYou = () => {
-	const [ pollCount, setPollCount ] = useState( 0 );
-	const selectedSiteId = useSelector( getSelectedSiteId );
-	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
-	const dispatch = useDispatch();
-
-	const isFetchingTransferStatus = useSelector( ( state ) =>
-		isFetchingAutomatedTransferStatus( state, selectedSiteId ?? 0 )
-	);
-
-	const isPluginInstalledDuringPurchase = useSelector( getIsPluginInstalledDuringPurchase );
-	const transferStatus = useSelector( ( state ) =>
-		getAutomatedTransferStatus( state, selectedSiteId ?? 0 )
-	);
-	const menuItems = useSiteMenuItems();
-	const isRequestingMenu = useSelector( getIsRequestingAdminMenu );
-	const { url: postsPageUrl } =
-		menuItems.find( ( { slug }: { slug: string } ) => slug === 'edit-php' ) ?? {};
-
-	const { url: yoastSeoPageUrl } =
-		menuItems.find( ( { slug }: { slug: string } ) => slug === 'wpseo_dashboard' ) ?? {};
-
-	const translate = useTranslate();
-
-	useEffect( () => {
-		if ( ! postsPageUrl || ! yoastSeoPageUrl ) {
-			selectedSiteId && dispatch( requestAdminMenu( selectedSiteId ) );
+const ThankYouContainer = styled.div`
+	.marketplace-thank-you {
+		margin-top: var( --masterbar-height );
+		img {
+			height: auto;
 		}
-	}, [ dispatch, postsPageUrl, selectedSiteId, yoastSeoPageUrl ] );
+	}
+`;
 
-	useEffect( () => {
-		if (
-			isPluginInstalledDuringPurchase &&
-			transferStatus !== 'complete' &&
-			! isFetchingTransferStatus
-		) {
-			setTimeout( () => {
-				selectedSiteId && dispatch( fetchAutomatedTransferStatus( selectedSiteId ) );
-				setPollCount( ( c ) => c + 1 );
-			}, 1500 );
-		}
-	}, [
-		isFetchingTransferStatus,
-		isPluginInstalledDuringPurchase,
-		pollCount,
-		setPollCount,
-		dispatch,
-		transferStatus,
-		selectedSiteId,
-	] );
+const MarketplaceThankYou = ( { productSlug } ): JSX.Element => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const plugin = useSelector( ( state ) => getPluginOnSite( state, siteId, productSlug ) );
+	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
 
-	/* TODO: Make all these items product-dependent */
 	const thankYouImage = {
-		alt: 'yoast logo',
-		src: yoastInstalledImage,
+		alt: 'success image',
```

Comment:
If we really want a text here, it should be translatable. However, I think this is just a decorative image as opposed to an informative one, and thus the `alt` property could be changed to simply
```suggestion
		alt: '',
```